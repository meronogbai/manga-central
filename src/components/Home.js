import { Link } from 'react-router-dom';
import styles from '../styles/Home.module.css';

const Home = () => (
  <main className={styles.home}>
    <div>
      <h1>Manga Central</h1>
      <Link to="/manga" className={styles.cta}>Top Manga</Link>
    </div>
  </main>
);

export default Home;
