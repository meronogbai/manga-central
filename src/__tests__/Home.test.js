import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Home from '../components/Home';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Home />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
