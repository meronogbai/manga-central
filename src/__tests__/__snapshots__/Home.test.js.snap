// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="home"
>
  <div>
    <h1>
      Manga Central
    </h1>
    <a
      className="cta"
      href="/manga"
      onClick={[Function]}
    >
      Top Manga
    </a>
  </div>
</main>
`;
